staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go

```diff
@@ -54,13 +54,51 @@ var (
 		},
 		[]string{"code", "method", "host"},
 	)
+
+	execPluginCertExpiration = k8smetrics.NewGauge(
+		&k8smetrics.GaugeOpts{
+			Name: "rest_client_exec_plugin_certificate_expiration_seconds",
+			Help: "Gauge of the expiry time of the auth exec plugin client " +
+				"certificate. The value is in seconds since January 1, 1970 UTC. " +
+				"If auth exec plugin client certificates are unused, gauge will remain at 0.",
+		},
+	)
```

Comment:
Should we rename this metric? The help text also could be updated.

Reply:
I think the name is appropriate but the help text has been updated.